cmd/mapcidr/main.go

```
@@ -252,10 +255,73 @@ func (options *Options) configureOutput() {
 	}
 }
 
-var options *Options
+var (
+	options *Options
+	// createdTempDirs stores directories created by this process
+	createdTempDirs []string
```

golangci-lint (2.13.2)

[error] var createdTempDirs is unused

(unused)